docs/platforms/kubernetes-cluster-addons/kubernetes_dashboard.md

```diff
@@ -0,0 +1,23 @@
+---
+hide:
+  - footer
+---
+
+
+## Introduction
+The Kubernetes Dashboard Web UI is available for deployed Kubernetes platforms on Azimuth. This provides a user-friendly way to administer a cluster.
+
+For usage information, see the [Kubernetes Docs](https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/#welcome-view).
+
+![Example of the Kubernetes Dashboard](../../assets/images/kubernetes_dashboard_example.png)
+
+### Accessing Kubernetes Dashboards
+Kubernetes Dashboard can be accessed via the link in the cluster details
+![Access the link in Details under Services](../../assets/images/accessing_kubernetes_dashboard.png){ loading=lazy }
+
+Kubernetes Dashboard can be enabled/disabled on an existing cluster using the green <img loading="lazy" class="off-glb" src="../../assets/images/update-button.svg" style="height:1em; vertical-align:middle;"> `Update` button in details, under the Cluster Addons section.
+
+Access can be granted through the [Identity Provider](../../identity_provider/identity_provider.md).
+
+!!! Info
+    Although access can be granted to Kubernetes Dashboard via Identity Provider, CLI (Kubectl) access via generating a Kubeconfig cannot. We recommend access to clusters be limited to users who already have admin access via your OpenStack project.</br>If access to other users is necessary, we recommend it is only to the Kubernetes Dashboard via the Identity Provider, as this can be revoked later. Access to the kubeconfig cannot be granted through azimuth to non-admin users, it can only be manually shared which we do not recommend.
```

Comment:
This took a few attempts to parse what you meant. Could we change it to something like:

> Access can be granted to Kubernetes Dashboard via Identity Provider only. The dashboard CLI cannot be accessed by generating a Kubeconfig.

Reply:
We recommend access to clusters be limited to users who already have admin access via your OpenStack project.

We should put why explicitly, i.e. `as users with access can create, delete or modify resources in your cluster(s)`